var/spack/repos/builtin/packages/libsakura/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Libsakura(CMakePackage):
+    """High-performance, thread-safe library compatible with C and C++
+    that is optimized for data analysis of astronomy and astrophysics.
+    """
+
+    homepage = "http://alma-intweb.mtk.nao.ac.jp/~sakura/"
```

Comment:
This URL does not appear to be valid or generally available.

Reply:
Thanks. I just changed it to https://alma-intweb.mtk.nao.ac.jp/~sakura/api/html/index.html and pushed an update. The new URL works for me, please let me know if you believe the new one to be invalid.